src/assets/style/module/controls/_controls.less

```diff
@@ -3,22 +3,36 @@
     display: inline-block;
 }
 
+#controls > .github-logo{
+    width: auto;
+}
+
+.github-logo {
+    img {
+        width: auto;
+        height: 20px;
+        margin-top: 4px;
+        padding: 4px 4px 8px 4px;
+    }
+}
+
+
 .control {
     margin: 0;
     transition: box-shadow 0.05s ease;
 }
 
-.control img {
+.control > img {
     opacity: 0.5;
```

Comment:
these two rules can be moved to the main `.github-logo-img` declaration, unless you have a specific reason to split them out?  if there is, please add a comment here to explain that reason.

Reply:
??